src/components/VotingPower/index.tsx

```diff
@@ -4,86 +4,110 @@ import 'tippy.js/dist/tippy.css'
 import Big from 'big.js'
 
 import { BNtoDecimal } from '@/utils/numerals'
-import { useVotingPower } from '@/hooks/query/useVotingPower'
 
 import tooltip from '@assets/utilities/tooltip.svg'
 
 import * as S from './styles'
 
 interface IVotingPowerProps {
-  userWalletAddress: string
+  totalVotingPower?: Big
+  currentVotingPower?: Big
+  minimalVotingPower?: Big
   yourVotingPowerInProposal?: Big
-  isMobile?: boolean
 }
 
 const VotingPower = ({
-  userWalletAddress,
-  yourVotingPowerInProposal,
-  isMobile
+  currentVotingPower,
+  minimalVotingPower,
+  totalVotingPower,
+  yourVotingPowerInProposal
 }: IVotingPowerProps) => {
-  const { data } = useVotingPower({ id: userWalletAddress })
-
   return (
-    <S.VotingPower isMobile={isMobile}>
-      <S.YourVotingPower>
-        <span>
-          {yourVotingPowerInProposal === undefined
-            ? 'your voting power'
-            : 'Snapshot voting power'}
-          {yourVotingPowerInProposal === undefined ? (
+    <S.VotingPower>
+      {currentVotingPower && !yourVotingPowerInProposal && (
+        <S.YourVotingPower>
+          <span>
+            your voting power
             <Tippy content="Voting power allows you to create and vote on proposals. To obtain voting power you need to stake your $KACY tokens.">
               <S.Tooltip tabIndex={0}>
-                <Image src={tooltip} alt="Explanation" width={14} height={14} />
+                <Image
+                  src={tooltip}
+                  alt="tooltip for voting power"
```

Comment:
The image is not a tooltip, the alt should describe what is on the image. Since it doesn't matter for the user that is using screenreader, you can add a `aria-hidden="true"` here. No aria is better than bad aria.

Reply:
What you can do is add a aria-label on the `S.Tooltip` and describe what this button does, instead